server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -880,6 +880,21 @@ public Metadata getMetadata() throws PathException {
     }
   }
 
+  /**
+   * Get devices info with given prefixPath.
+   *
+   * @return A HashSet instance which stores devices info with given prefixPath.
```

Comment:
Remove the imprecise name "info" or change it to "device names(or Ids)".

Reply:
Fix it.